File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodeSectorProperties.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodeSectorProperties (
    _travelTimeReachable: Option[ResponseTravelTimeStatistics],
    _travelTimeAll: Option[ResponseTravelTimeStatistics],
    _coverage: Option[Double]
)
object ResponseTimeFilterPostcodeSectorProperties {
    def toStringBody(var_travelTimeReachable: Object, var_travelTimeAll: Object, var_coverage: Object) =
        s"""
        | {
        | "travelTimeReachable":$var_travelTimeReachable,"travelTimeAll":$var_travelTimeAll,"coverage":$var_coverage
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseTimeFilterPostcodeSectorProperties.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseTravelTimeStatistics

module ResponseTimeFilterPostcodeSectorProperties = 

  //#region ResponseTimeFilterPostcodeSectorProperties


  type ResponseTimeFilterPostcodeSectorProperties = {
    TravelTimeReachable : ResponseTravelTimeStatistics;
    TravelTimeAll : ResponseTravelTimeStatistics;
    Coverage : double;
  }
  //#endregion
  